lib/components/artist/artworks/grid.js

```diff
@@ -118,7 +118,7 @@ class ArtworksGrid extends React.Component {
       // There are artworks without images and other ‘issues’. Like Force we’re just going to reject those for now.
       // See: https://github.com/artsy/eigen/issues/1667
       //
-      if (artwork.image) {
+      if (artwork.image && artwork.partner) {
```

Comment:
Should I leave this here until we figure out what (if anything) needs to be changed on the Artwork page?